model_draft_transactions_test.go

```diff
@@ -45,15 +45,17 @@ func TestDraftTransaction_newDraftTransaction(t *testing.T) {
 	t.Run("valid config", func(t *testing.T) {
 		expires := time.Now().UTC().Add(defaultDraftTxExpiresIn)
 		draftTx := newDraftTransaction(
-			testXPub, &TransactionConfig{}, New(),
+			testXPub, &TransactionConfig{
+				FeeUnit: chainstate.MockDefaultFee,
```

Comment:
Isn't it set by test config of bux?
Maybe we should use here other value to check if it is applied?

Reply:
We can write a test case where we test the `newDraftTransaction` in isolation (without client) and check whether it panics when we do not provide `FeeUnit` in the `TransactionConfig`

Reply:
Let's leave it like it is for now.
But for the future, for me it is very weird, that we're testing function without "client", and actually this function partially depends on this client.